cookbooks/planet/templates/default/planetdump.erb

```diff
@@ -2,8 +2,8 @@
 
 # DO NOT EDIT - This file is being maintained by Chef
 
-# Exit on error
-set -e
+# Exit on errors
+set -euo pipefail
```

Comment:
-o pipefail is generally not recommended to be enabled globally. It often results in unexpected and weird pipeline behavior. It's okay for pipelines to return non-zero status codes.